sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -57,8 +54,20 @@ src_prepare() {
 	gperf -o -i 7 --ignore-case -C -k 1-4,6,9,\$ -H keyword_hash -N check_identifier -t ./lexor_keyword.gperf > lexor_keyword.cc || die
 }
 
+src_compile() {
+	default
+}
+
 src_install() {
 	local DOCS=( *.txt )
+	# Fix build fail problem when using large job number, make it parallel safe
+	# See also https://github.com/gentoo/gentoo/pull/14096
+	# and https://github.com/vowstar/vowstar-overlay/issues/1
+	# NOTE: this is only a workaround for upstream parallel build bug
+	# this fix made by minux https://github.com/minux
+	# https://github.com/steveicarus/iverilog/pull/294
+	# Due to this issue, we must run ``emake installdirs`` before ``default``
```

Comment:
Too much text here, the first line and upstream issue link is enough. Ie, 
```
# Default build fails with parallel jobs, 
# https://github.com/steveicarus/iverilog/pull/294
```
That's enough information.

Reply:
Thank you! I've applied it to both 10.3 and 9999 ebuild